### PR TITLE
FE-1641: Add CSP header

### PR DESCRIPTION
**Pull request by theus**
- Add CSP Headers
- Created [developers-sparkpost-com](https://sentry.io/organizations/sparkpost/projects/developers-sparkpost-com/?project=6351005&statsPeriod=14d) project on Sentry to monitors CSP violations

image

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -12,6 +12,55 @@
...
+      *;
+    child-src
+      'self';
+    img-src
+      'self'
+      data:
+      *;
+    font-src
+      'self'
+      *.cloudfront.net;
```
Comment (shrirupa) [inline netlify.toml:44]: should we include a * here ?

**Comment by shrirupa** [comment]
Seeing this error while going over the deploy preview 
``` The Content Security Policy directive 'upgrade-insecure-requests' should be empty, but was delivered with a value of '1'. The directive has been applied, and the value ignored ```

**Comment by shrirupa** [comment]
Another csp error 🙈 
```
Refused to send form data to 'https://www.facebook.com/tr/' because it violates the following Content Security Policy directive: "form-action 'self'".

```

**Comment by shrirupa** [comment]
looks like since we should probably send out the report-only header before we make the cutover to csp.

File: netlify.toml (hunk 1 above)
Comment (theus) [reply netlify.toml:44]: this CloudFront address it's from netlify (I think) so, for some reason, the subdomain could change? 🤔

Comment (shrirupa) [reply netlify.toml:43]: sorry, I meant font-src * self *.cloudfront.net

Comment (theus) [reply netlify.toml:43]: oh, IDK. When I was checking, the only domain that load fonts was this CloudFront 🤔

**Review by shrirupa — APPROVED** [review]

**Pull request merged**